profiles/models.py

```diff
@@ -144,3 +145,21 @@ def __str__(self):
     def pretty_printed_student_id(self):
         """pretty prints the student id for easy display"""
         return "MMM{0:06}".format(self.student_id) if self.student_id else ""
+
+
+class Education(models.Model):
+    DEGREE_CHOICES = (
+        (DOCTORATE, 'Doctorate'),
+        (MASTERS, "Master's or professional degree"),
+        (BACHELORS, "Bachelor's degree"),
+        (ASSOCIATE, "Associate degree"),
+        (HIGH_SCHOOL, "High school"),
+        (OTHER_EDUCATION, "Other education"),
+    )
+    profile = models.ForeignKey(Profile, related_name='education')
+    degree_name = models.CharField(max_length=30, choices=DEGREE_CHOICES)
+    graduation_date = models.DateField()
+    subject = models.CharField(max_length=30)
```

Comment:
I don't think it's necessary. For the profile we needed it most places because the most of profile starts off blank, but we're only creating `Education` objects when the user is editing their own profile.